.gitignore

```diff
@@ -22,3 +23,6 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 publish.bat
+
+# intellij configs
+.idea
```

Comment:
> In the future, this type of change should be done in a separate PR, since it's unrelated to what you're fixing in this one. It's better to have more, smaller PRs, vs adding complexity and scope to existing ones.

noted thanks